static/lang/re-en.json

```diff
@@ -3358,6 +3358,10 @@
                 "RobeOfTheArchmagi": {
                     "VsArcaneSpells": "{item} (vs. Arcane Spells)"
                 },
+                "RootBoots": {
+                    "GripGround": "Root Boots - Grip the Ground",
+                    "WoodenSurfaceLabel": "Root Boots - Wooden Surface"
+                },
```

Comment:
I've been using em-dashes for labels like these so far. Which way would you prefer so we can standardise?

Reply:
Good question. @TikaelSol do you have a preference for dash type?

Reply:
I'm indifferent, but since it's a localization thing it can be changed later if needed.

Reply:
Switched to emdash.